tests/test_card.py

```diff
@@ -26,26 +24,26 @@ def test_relaycard():
         assert rly.is_initialized is True
         assert rly.card_count == 255
 
-        mock_serial_instance.read.return_value = b"\xFD\x00\x00\xFD"
+        mock_serial_instance.read.return_value = b"\xfd\x00\x00\xfd"

```

Comment:
Why was this chance required? Why is
it now lowercased?

Reply:
I think this done by ruff --format 😅